### PR TITLE
DS-10 | New buttons

### PR DESCRIPTION
**Pull request by vforge**
https://app.zeplin.io/project/55f9910b3bdc5fad1481b004/screen/5984fa43ec0e71950a11f1ab

New buttons & (very) minor cleanup

### CODE REVIEW

File: style-guide/styles/wds-components/_buttons.scss [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-	padding: 14px $wds-button-padding-horizontal;
+	padding: 7px 18px;
 	text-decoration: none;
 	text-transform: uppercase;
 	transition-duration: $wds-button-transition-duration;
 	transition-property: background-color, border-color, color;
 	vertical-align: top;
 	-webkit-appearance: none;
-	-webkit-border-radius: 0;
+	-webkit-border-radius: $wds-rounded-corner-radius;
```
Comment (latata) [inline style-guide/styles/wds-components/_buttons.scss]: is it still needed?

**Comment by vforge** [comment]
@latata I fixed both issues.

**Comment by hakubo** [comment]
LGTM :thumbsup:
what do we need to get that merged?

**Comment by vforge** [comment]
@hakubo It's dne

**Pull request merged**